### ISSUE TITLE
`git hf feature submit` errors out if not using Github

### ISSUE THREAD

**Issue by pgporada** [datasift/gitflow#87]
We're using Hubflow with Bitbucket being our DVCS. When trying to run the following command
```
git hf feature submit
```
we are returned
```
The remote repo for this branch, 'origin' is not a GitHub repository.
```

I believe this is due to the following lines.
https://github.com/mborsuk/hubflow/blob/develop/hubflow-common#L199-L206
https://github.com/mborsuk/hubflow/blob/develop/hubflow-common#L535-L541

**Comment by stuartherbert** [comment]
Yeah. That's the only part of the Hubflow extension that actually knows about Github - it uses the Github API. I'm planning on removing this command unless there's a lot of users out there who rely on it.

**Comment by pgporada** [comment]
Stuart,

My team of ~12 people are wanting to use the feature but with BitBucket instead of Github. Is it possible to have some sort of check to see if the origin is Github or BitBucket and have the feature submit act accordingly?

**Comment by stuartherbert** [comment]
I'll take a look and see. Give me a little bit of time to investigate; I am not a BitBucket user.

**Comment by EslamElHusseiny** [comment]
@stuartherbert @pgporada  I've addressed bitbucket support for "git hf feature submit" command here https://github.com/datasift/gitflow/pull/88

**Comment by bitwiseman** [comment]
@stuartherbert - my team uses the automatic pull request creation.  Can you make the check optional, so people can turn it off if they're not using github?

**Comment by EslamElHusseiny** [comment]
@bitwiseman what kind of git servers you're using rather than github and bitbucket ?

**Comment by ladyrassilon** [comment]
I have to say, creation of pull requests by command line are a really nice feature...

**Comment by tommynsong** [comment]
+1 for optional
we are using Github Enterprise
